source/Nevermore/Util/DataModificationQueryBuilder.cs

```diff
@@ -342,7 +342,7 @@ CommandParameterValues GetDocumentParameters(Func<DocumentMap, object> allocateI
             switch (mapping.JsonStorageFormat)
             {
                 case JsonStorageFormat.TextOnly:
-                    result[$"{prefix}{JsonVariableName}"] = serializer.SerializeText(document, mapping);
+                    result[$"{prefix}{JsonVariableName}"] = serializer.SerializeText(document, mapping).ReadToEnd();
```

Comment:
Can't rewind textreaders, so as a work around for the POC I just output the string. Will need to come up with something better if we go ahead with this